doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md

```diff
@@ -170,6 +170,17 @@ Jumpers on the CC3200 board should be connected as shown below.
 
 ![Jumpers](attach/jumpers_debug_mode_400.png)
 
-To run an application, remove SOP2 and J8 jumpers, and then connect J2 and J3 jumpers.
+Then install UniFlash for [Linux](http://software-dl.ti.com/dsps/forms/self_cert_export.html?prod_no=uniflash_3.4.1.00012_linux.tar.gz&ref_url=http://software-dl.ti.com/ccs/esd/uniflash/) or [Windows](http://software-dl.ti.com/dsps/forms/self_cert_export.html?prod_no=uniflash_3.4.1.00012_win32.zip&ref_url=http://software-dl.ti.com/ccs/esd/uniflash/).
```

Comment:
The Item is called **Flashing (Windows only)**, but you're describing both linux and windows flashing